ironfish/src/rpc/routes/chain/snapshotChain.test.ts

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { Assert } from '../../../assert'
+import { getBlockSize, writeBlock } from '../../../network/utils/block'
+import { useMinerBlockFixture } from '../../../testUtilities'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+import { SnapshotChainStreamResponse } from './snapshotChain'
+
+describe('Route chain/snapshotChainStream', () => {
+  const routeTest = createRouteTest()
+
+  describe('with no maxBlocksPerChunk parameter', () => {
+    it('correctly returns a serialized list of blocks', async () => {
+      const { chain, strategy } = routeTest
+      await chain.open()
+      strategy.disableMiningReward()
+
+      const genesis = await chain.getBlock(chain.genesis)
```

Comment:
Same for this, seems like it is just testing our test utils

Reply:
I think this is fine enough to leave as-is for this PR.